README.md

```diff
@@ -1,24 +1,90 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Car Rentals Application UI
 
-Things you may want to cover:
+This project is made for the car rentals. This website provides a platform for people to easily check available cars, choose favourite cars and rent cars for specific period of time.
 
-* Ruby version
+Main features of this website are:
 
-* System dependencies
+- Users can see details of all the Cars.
+- Users can rent cars for specific period of time.
+- Users can add cars and delete cars.
 
-* Configuration
+## Built With
 
-* Database creation
+- Programming Languages: HTML, CSS, JavaScript.
+- Frameworks: React, Redux and Bootstrap.
+- Code Editor: VS Code.
+- Technologies: Git, Gitflow, Linters, Github, PostgreSql
 
-* Database initialization
+## Kanban Board:
+Kanban Board [link](https://github.com/users/efoosag/projects/10)
+**Initial state of the Kanban Board:**
+![final capstone kanban board](Start Final Capstone.png)
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+## Getting Started
 
-* Deployment instructions
+To get a local copy up and running follow these simple example steps.
 
-* ...
+### Prerequisites
+
+- A web browser like Google Chrome.
+- A code editor like Visual Studio Code with Git and nodejs.
+- Have `visual-studio code` or any other code editor installed on your computer.
+
+### Setup
+
+- In order to get a copy of this project you need to clone it from `https://github.com/efoosag/car-rental-app-backend.git `
+- The link to the frontend file is  `https://github.com/efoosag/car-rental-app.git`t`
+
+
+### Install and start
+
+- Run the command bellow in your terminal to get all required files
+
+```
+npm install
+```
+
+```
+npm start
+```
```

Comment:
- [ ] These are not the correct instructions to setup and install the backend project.  Kindly include how to set up and run the setup project.  Don't forget the DB commands like rails db:create. 👍